packages/treasury/src/vaultManager.js

```diff
@@ -32,28 +40,34 @@ export function makeVaultManager(
   runMint,
   collateralBrand,
   priceAuthority,
-  rates,
+  rateManager,
   reallocateReward,
   timerService,
-  loanParams,
+  loanParamManager,
   liquidationStrategy,
 ) {
   const { brand: runBrand } = runMint.getIssuerRecord();
 
   const shared = {
     // loans below this margin may be liquidated
     getLiquidationMargin() {
-      return rates.liquidationMargin;
+      return rateManager.lookup(LIQUIDATION_MARGIN_KEY);
     },
     // loans must initially have at least 1.2x collateralization
     getInitialMargin() {
-      return rates.initialMargin;
+      return rateManager.lookup(INITIAL_MARGIN_KEY);
     },
     getLoanFee() {
-      return rates.loanFee;
+      return rateManager.lookup(LOAN_FEE_KEY);
```

Comment:
the manager is the guy who can change them, not report them, so I don't think we want that naming.